packages/blockchain-tools/src/interfaces/Web3Server.ts

```diff
@@ -62,18 +69,31 @@ export default class Web3Server extends ChainBaseClass {
     const sign = await this.web3.eth.accounts.signTransaction(tx, walletSeed)
     const info = await this.web3.eth.sendSignedTransaction(sign.rawTransaction)
     console.log("INFO", info)
-    const hasLogs = info.logs.length > 0
+    //const hasLogs = info.logs?.length > 0
+    //const hasTopics = hasLogs && info.logs[0]?.topics?.length > 0
     let tokenNum = ""
-    if (hasLogs) {
+    if(info.logs?.length > 0) {
       console.log("LOGS.0", JSON.stringify(info?.logs[0].topics, null, 2))
-      console.log("LOGS.1", JSON.stringify(info?.logs[1].topics, null, 2))
-      tokenNum = ` #${Number.parseInt(Buffer.from(_get(info, "logs.0.topics.3", Buffer.alloc(0))).toString("hex"), 16)}`
+      //console.log("LOGS.1", JSON.stringify(info?.logs[1].topics, null, 2))
+      //tokenNum = ` #${Number.parseInt(Buffer.from(_get(info, "logs.0.topics.3", Buffer.alloc(0))).toString("hex"), 16)}` // Doesn't work as expected
+      //const nftSeq = Number.parseInt((info?.logs[0]?.topics[3] || 0).toString(),16)
+      const nftSex = getObjectValue(info, 'logs.0.topics.3')
```

Comment:
Looks like you can just use a nullish check here
```suggestion
      const nftSex = info?.logs?.[0]?.topics?.[3]
```

Reply:
For some reason that never worked, I tried many different ways and lodash wasn't converting types correctly